pkgs/tools/typesetting/tex/texlive/combine.nix

```
@@ -299,7 +302,12 @@ in (buildEnv {
       )
     fi
   ''
-    + bin.cleanBrokenLinks
+    + bin.cleanBrokenLinks +
+  # Get rid of all log files. They are not needed, but take up space
+  # and render the build unreproducible by their embedded timestamps.
+  ''
+    find $out -name '*.log' -delete
```

```suggestion
    find "$TEXMFSYSVAR/web2c" -name '*.log' -delete
```
Merely as precaution.

---

Good idea, did so in incoming force push!